### ISSUE TITLE
lodash dependency for twilio

### ISSUE THREAD

**Issue by copremesis** [serverless/examples#622]
```{
    "errorType": "Runtime.ImportModuleError",
    "errorMessage": "Error: Cannot find module 'lodash'\nRequire stack:\n- /var/task/node_modules/twilio/lib/webhooks/webhooks.js\n- /var/task/node_modules/twilio/lib/index.js\n- /var/task/handler.js\n- /var/runtime/UserFunction.js\n- /var/runtime/index.js",
    "trace": [
        "Runtime.ImportModuleError: Error: Cannot find module 'lodash'",
        "Require stack:",
        "- /var/task/node_modules/twilio/lib/webhooks/webhooks.js",
        "- /var/task/node_modules/twilio/lib/index.js",
        "- /var/task/handler.js",
        "- /var/runtime/UserFunction.js",
        "- /var/runtime/index.js",
        "    at _loadUserApp (/var/runtime/UserFunction.js:100:13)",
        "    at Object.module.exports.load (/var/runtime/UserFunction.js:140:17)",
        "    at Object.<anonymous> (/var/runtime/index.js:43:30)",
        "    at Module._compile (internal/modules/cjs/loader.js:999:30)",
        "    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)",
        "    at Module.load (internal/modules/cjs/loader.js:863:32)",
        "    at Function.Module._load (internal/modules/cjs/loader.js:708:14)",
        "    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12)",
        "    at internal/main/run_main_module.js:17:47"
    ]
}```

trying https://github.com/serverless/examples/tree/master/aws-node-twilio-send-text-message
but the README only describes how to setup the serverless portion
yet the nodeJS stuff is quite broken or not obviously documented

**Comment by pgrzesik** [comment]
Hello @copremesis :wave: Thanks for reporting. Unfortunately, that example is quite outdated, we'd be more than happy to accept a contribution that updates it :raised_hands:

**Comment by slaytr** [comment]
@pgrzesik PR created containing the fix for this at #633

**Comment by pgrzesik** [comment]
Thank you @slaytr for fixing it :bow:
